### DISCUSSION TITLE
idiot question: how do I install this locally?

### DISCUSSION THREAD

**Discussion by seatrout** [akosbalasko/yarle#200] [Q&A] [answered]
I have got this running successfully with the no-installation method, which seems to pull all the needed files down before starting up. How do I install a local copy which would skip that step?

**Comment by akosbalasko** [comment]
Hi @seatrout , you can clone this repository to have all the code of Yarle, and then it can be built locally by typing `npm i` and then `npm run build`. After you have installed Node of course. 

Cheers,
akos

**Reply by seatrout** [reply]
Ah. Thanks. But the online method will always get the latest release, right? Whereas this one will only have whatever branch I clones ...

**Reply by akosbalasko** [reply, marked as answer]
yes, the online one gets the latest release. But you can clone the repo and checkout the master, you can get the latest changes if you keep your local branch updated by `git pull` , independently from the change was released or not. So in this sense it provides a more fresh version of the code.
